### PR TITLE
HEMTT and Github Actions CI

### PR DESCRIPTION
**Pull request by Dahlgren**

### CODE REVIEW

File: hemtt.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name = "Gruppe Adler Mod"
+prefix = "grad_"
```
Comment (DerZade) [inline hemtt.toml:2]: Is this underscore necessary?

Comment (Dahlgren) [reply hemtt.toml:2]: No, it shouldn't be there :D

**Comment by Dahlgren** [comment]
For some reason the include files don't work on GitHub Actions CI, https://github.com/Dahlgren/gruppe_adler_mod/runs/565984268

But they work locally for me 🤔

**Comment by AdlerSalbei** [comment]
That is weird 🤔

**Comment by Dahlgren** [comment]
Its because I only tried on macOS and Windows which both use a case insensitive file system by default 🙃 

And now its green, https://github.com/Dahlgren/gruppe_adler_mod/runs/594818431

Should I leave the armake stuff like it is? Any changes wanted on Travis CI?

**Comment by Dahlgren** [comment]
Should I bundle some version of hemtt, preferably the same as CI uses, just like armake in this repo or hemtt in grad trenches?

**Comment by Fusselwurm** [comment]
> Should I bundle some version of hemtt, preferably the same as CI uses, just like armake in this repo or hemtt in grad trenches?

I'm personally not a fan of binaries in repositories, and especially not standard build tools. 
Lets wait for @DerZade to chime in.

**Comment by DerZade** [comment]
@Fusselwurm @Dahlgren it depends if anyone ever wants to build this locally of if downloadable build artifacts for every push are enough 😅 (for example see [here](https://github.com/Dahlgren/gruppe_adler_mod/runs/594818431) in the top right corner)
I don't work a lot with this repo so for me artifacts would be more than enough. But I don't know what our top contributors @AdlerSalbei and @McDiod think about that 😉

**Comment by McDiod** [comment]
I'll need an easy way to build this locally. If there are ways other than including the hemtt binary I'm open for it, but emphasis is on *easy*.

**Comment by Fusselwurm** [comment]
> I'll need an easy way to build this locally. If there are ways other than including the hemtt binary I'm open for it, but emphasis is on _easy_.

install hemtt on your system, navigate to project, call hemtt <whatever> . easy.
(does it have an installer in windows, or do we need to put it into the PATH ?)

**Comment by McDiod** [comment]
`hemtt build` works, but it seems to just pack all subfolders in `addons` into `.pbo`, not the entire mod. `hemtt build release` gives me this:

```
$ hemtt build release
HEMTT 0.7.6
Environment: dev

warning: .gitignore is missing recommended value `releases/*`
warning: .gitignore is missing recommended value `*.biprivatekey`
warning: .gitignore is missing recommended value `keys/*`
warning: .gitignore is missing recommended value `.hemtt/local`
warning: deprecated value `keyname` in `hemtt.toml` - use `key_name`
warning: deprecated value `signame` in `hemtt.toml` - use `authority`
```

**Comment by Dahlgren** [comment]
Those warnings should be fixed now

**Pull request merged**